src/clj/fluree/db/util/core.cljc

```diff
@@ -206,7 +206,7 @@
   (reduce-kv
     (fn [acc k v]
       (if (keyword? k)
-        (assoc acc (name k) v)
+        (assoc acc (subs (str k) 1) v)
```

Comment:
If we're keeping the namespace of namespaced keywords I think the docstring for this function should be updated as well.

Reply:
I'll revert this back to its original state - I needed this behavior for a feature and realized it would still work with the other uses.

But then I removed the feature, so the change is no longer needed.